Auth/Auth.Platform.Android/Security/Checks/NonRootedCheck.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Android.Content;
+
+namespace AeroGear.Mobile.Security.Checks
+{
+    /// <summary>
+    /// This is just a stub implementation. Fill with the real implementation.
+    /// </summary>
+    public class NonRootedCheck : ISecurityCheck
+    {
+        private const string NAME = "Rooted Check";
+
+        private readonly Context context;
+
+        public NonRootedCheck(Context ctx)
+        {
+            this.context = ctx;
+        }
+
+        public string GetId()
+        {
+            return typeof(NonRootedCheck).FullName;
+        }
+
+        public string GetName()
+        {
+            return NAME;
+        }
+
+        public SecurityCheckResult Test()
```

Comment:
maybe just call it "Check"?

Reply:
I called it 'test' for coherency with the Android SDK (there it is called `test`).
Do we want to change it here?

Reply:
Yes, the JS SDK is using "check". It's a bit inconsistent at the moment.

Let's use "check" in Xamarin as well, and we can update the Android SDK later on to keep them consistent.